benchmark/net/net-c2s-cork.js

```diff
@@ -0,0 +1,96 @@
+// test the speed of .pipe() with sockets
+
+var common = require('../common.js');
+var PORT = common.PORT;
+
+var bench = common.createBenchmark(main, {
+  len: [4, 8, 16, 32, 64, 128, 512, 1024],
+  type: ['buf'],
```

Comment:
Is there a reason to keep the [other encoding stuff](https://github.com/mcollina/node/commit/3e6220c25e4c7868a36c9d356effdbcc6512a795#diff-3731609dc6785b4cd2208d74439c4542R28) if we only have buffers here?